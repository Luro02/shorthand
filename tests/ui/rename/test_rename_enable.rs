use shorthand::ShortHand;

#[derive(ShortHand)]
#[shorthand(enable(rename))]
struct Command {
    value: String,
}

fn main() {}
