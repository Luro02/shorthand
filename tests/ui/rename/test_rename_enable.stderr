error: unexpected field `rename`
 --> $DIR/test_rename_enable.rs:4:20
  |
4 | #[shorthand(enable(rename))]
  |                    ^^^^^^
